scripts/pipeline/orthology_benchmark.py

```diff
@@ -307,8 +310,32 @@ def run_orthology_tools(input_dir: str, orthofinder_params: str) -> None:
         raise RuntimeError(msg) from exc
 
 
-def prep_input_for_goc():
-    """Docstring"""
+def extract_orthologs(res_dir: str, species_key1: str, species_key2: str) -> List[Tuple[str, str]]:
+    """Returns a list of putative orthologous pairs inferred by OrthoFinder for a specified pair
+    of species identificators used in the OrthoFinder analysis.
```

Comment:
Ideally the summary line of a multi-line docstring should fit on one line. One way to achieve this would be to summarise the function here and add a `Returns` section into which you could put the details of the return value that are currently given here.